GatherBuddy/AutoGather/AutoGather.Artisan.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Dalamud.Plugin.Ipc.Exceptions;
+using ECommons.Throttlers;
+using ECommons;
+using GatherBuddy.Plugin;
+using System;
+using ECommons.DalamudServices;
+
+namespace GatherBuddy.AutoGather
+{
+    public partial class AutoGather
+    {
+        internal void PauseArtisan()
+        {
+            try
+            {
+                if (IsArtisanOperating())
+                {
+                    WasArtisanPaused = true;
+                    GatherBuddy.Log.Information("Paused Artisan");
+                    Artisan_IPCSubscriber.SetStopRequest(true);
+                    TaskManager.Enqueue(() => AutoStatus = "Waiting for Artisan to stop crafting...");
+                    TaskManager.Enqueue(() => !IsCrafting, 240000);
+                    TaskManager.Enqueue(() =>
+                    {
+                        if (IsCrafting)
+                        {
+                            Communicator.Print("[GatherBuddy Reborn] Requested Artisan to stop crafting but crafting state has gone longer than 4 minutes...Aborting Auto Gather");
+                            AbortAutoGather();
+                        }
```

Comment:
Need second opinion if handling it like this is intended, since I think it is in GBR's best interest to not interrupt the user's ongoing craft.